### DISCUSSION TITLE
Why is selection of "/" as database not possible?

### DISCUSSION THREAD

**Discussion by mgw2013** [cboxdoerfer/fsearch#444] [Q&A] [answered]
See subject!

**Comment by cboxdoerfer** [comment]
It is possible:
![image](https://user-images.githubusercontent.com/6108388/202498287-bc8bef5b-0356-4339-91f5-30b0fc84191a.png)

Can you be more specific where exactly the issue is?

**Comment by mgw2013** [comment]
Uhhh... 
```
$ uname -a
Linux MW-TPW520 5.15.0-52-generic #58~20.04.1-Ubuntu SMP Thu Oct 13 13:09:46 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
$ inxi -zxxxS
System:    Kernel: 5.15.0-52-generic x86_64 bits: 64 compiler: N/A Desktop: Cinnamon 5.2.7 wm: muffin 5.2.1 
           dm: LightDM 1.30.0 Distro: Linux Mint 20.3 Una base: Ubuntu 20.04 focal
$ fsearch -v
FSearch 0.1beta4
```
Can only select a folder under root but not root itself.

Hold on a sec. - will purge fsearch-trunk and install "official" flatpack via mintinstall...
 
Unchanged! But now I don't have a CLI anymore... :-( 
Now on 0.2.2 (flathub-stable)

**Reply by cboxdoerfer** [reply]
Is there some kind of error message? At what step does it fail to add `/`? It also works perfectly fine for me with the Flathub version.

**Reply by cboxdoerfer** [reply]
Also why does the CLI of the Flatpak not work for you? I haven't noticed any issues with it.

**Reply by mgw2013** [reply]
> Is there some kind of error message? At what step does it fail to add `/`? It also works perfectly fine for me with the Flathub version.

Nope. "/" is simply not offered as selectable item - fsearch calls Nemo for selection

**Reply by cboxdoerfer** [reply]
That's an issue with the file chooser then, which FSearch has no control over. It works fine for me with the GTK, Qt and KDE file choosers. Can you post a screenshot of the file chooser on your system when you try to select `/`?

**Reply by mgw2013** [reply]
Sure: Here we go: 
![Bildschirmfoto vom 2022-11-17 19-13-05](https://user-images.githubusercontent.com/5862935/202525169-23219ec7-8343-4d64-ab52-5e607c9c4caa.png)

**Reply by cboxdoerfer** [reply]
That's the default GTK file chooser. There are a couple of ways how to select `/` with it, which should work:

* Click on `Andere Orte` -> `Rechner` -> `Auswählen`
* or hit `Ctrl-l` type `/` and hit `Enter`
* or in your screenshot click on `home` while holding `Ctrl`, then select `Auswählen`

**Reply by mgw2013** [reply]
> * Click on `Andere Orte` -> `Rechner` -> `Auswählen`

This is how arrived at this screen... not working

Ctrl-L did the trick - suggest to mention it somewhere. Txs!

**Reply by mgw2013** [reply]
So what happend to the CLI... lemme reboot first...

**Reply by mgw2013** [reply]
Sorry....
```
$ fsearch

Der Befehl 'fsearch' wurde nicht gefunden, meinten Sie:

  Befehl 'pfsearch' aus dem deb pftools (3+dfsg-3)
  Befehl 'esearch' aus dem deb ncbi-entrez-direct (12.0.20190816+ds-1)
  Befehl 'csearch' aus dem deb codesearch (0.0~hg20120502-3)
  Befehl 'vsearch' aus dem deb vsearch (2.14.1-3build1)

Versuche: sudo apt install <deb name>
```

**Reply by cboxdoerfer** [reply]
That's not how you launch Flatpaks from the command line, unless you create a shell alias for that. By default it works like that:

`flatpak run io.github.cboxdoerfer.FSearch`

There you can append all the supported command line options, e.g.:

`flatpak run io.github.cboxdoerfer.FSearch --version`

**Reply by mgw2013** [reply]
I forgot that this is exactly why I'm not in favor of flatpack... Txs for ur help!

**Reply by cboxdoerfer** [reply]
No problem. Since you seem to be using Linux Mint you can of course use the stable PPA for native packages without any of the Flatpak limitations.

**Comment by mgw2013** [comment, marked as answer]
Done already! Thank you for this nice tool!


-------- Ursprüngliche Nachricht --------
Von: "Christian Boxdörfer" ***@***.***>
Gesendet: 18. November 2022 18:25:12 MEZ
An: cboxdoerfer/fsearch ***@***.***>
CC: mgw2013 ***@***.***>, Author ***@***.***>
Betreff: Re: [cboxdoerfer/fsearch] Why is selection of "/" as database not possible? (Discussion #444)

No problem. Since you seem to be using Linux Mint you can of course use the stable PPA for native packages without any of the Flatpak limitations.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/cboxdoerfer/fsearch/discussions/444#discussioncomment-4179167
You are receiving this because you authored the thread.

Message ID: ***@***.***>

Mit freundlichen Grüßen
Michael Weigelt
